services/grpc/internal/transformations/transformations.go

```diff
@@ -0,0 +1,91 @@
+package transformations
+
+const (
+	REPLACE = iota
+	TRANSCLUDE
+	DRAG
+	HIGHLIGHT
+)
+
+type FxnArgs struct {
+	FxnId        int
+	OffsetStart1 int
+	OffsetEnd1   int
+	OffsetStart2 int
+}
+
+// NOTE: How to get all functions to conform to signature: Call(string, string, *FxnArgs) (string, string, *FxnArgs) ?
+func Call(region0, src0 string, args *FxnArgs) (string, string, *FxnArgs) {
```

Comment:
Any reason the source was removed from the args? I thought that was an improvement.

Reply:
https://github.com/sourcegraph-xyz/sg/pull/5/files#diff-bcd403d2b86b5db40576a49ba6e877befe9d5485b8b96c4bafaec99575b73cd3R37

Reply:
Fixed, the changes are correct in the public-api package, just need to delete these old files